docs/changelog.md

```diff
@@ -82,6 +82,12 @@ For more information on features and bug fixes in 0.6.0, see the GitHub mileston
 
 #### Upgrade Instructions
 
+##### Experimental support for Amazon Keyspaces
+
+[Amazon Keyspaces](https://aws.amazon.com/keyspaces/) is a serverless managed Apache Cassandra-compatible
+database service provided by Amazon. See [Deploying on Amazon Keyspaces](https://docs.janusgraph.org/storage-backend/cassandra/#deploying-on-amazon-keyspaces-experimental)
```

Comment:
[This link](https://docs.janusgraph.org/storage-backend/cassandra/#deploying-on-amazon-keyspaces-experimental) currently does not have a Deploying on Amazon Keyspaces section. I've filed https://github.com/JanusGraph/docs.janusgraph.org/issues/5 which I hope might help us this.

Reply:
Closed the issue. The correct link is: https://docs.janusgraph.org/master/storage-backend/cassandra/